Bio/Phylo/TreeConstruction.py

```diff
@@ -374,38 +374,38 @@ class DistanceCalculator(object):
     --------
     >>> from Bio.Phylo.TreeConstruction import DistanceCalculator
     >>> from Bio import AlignIO
-    >>> aln = AlignIO.read(open('Tests/TreeConstruction/msa.phy'), 'phylip')
+    >>> aln = AlignIO.read(open('./TreeConstruction/msa.phy'), 'phylip')
```

Comment:
Could you just use ``'TreeConstruction/msa.phy'`` (no need for the leading dot, it is unnecessary and potentially confusing)

Reply:
Amended.